notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4446,6 +4446,55 @@ fear of commitment [Clementine accused Joel of this shortcoming.]
 love triangle [Harold, his wife, and the receptionist Mary, but Mary latter having wiped her memories of it.]
 
 
+movie: Immortal (2004)
+======================
+
+:: Title
+Immortal
+
+:: Description
+Immortal (French: Immortel, ad vitam) is a 2004 English language French
+live-action and animated science fiction film co-written and directed by Enki
+Bilal and starring Linda Hardy, Thomas Kretschmann and Charlotte Rampling. It
+is loosely based upon Bilal's comic book La Foire aux immortels (The Carnival
+of Immortals).
+
+Synopsis: The Ancient Egyptian god Horus, who is evidently a space alien,
+seeks to procreate with a human female, after being stripped of his
+immortality by his fellow gods.
+
+:: Date
+2004-03-24
+
+:: References
+https://en.wikipedia.org/wiki/Immortal_(2004_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+coping with mortality [The god/alien Horus was determined to mate with a human female after being stripped of his immortality. Having offspring was presumably his way of coping with the reality that he would eventually die.]
```

Comment:
What about:

https://www.themeontology.org/theme.php?name=the%20quest%20for%20immortality

https://www.themeontology.org/theme.php?name=immortality

https://www.themeontology.org/theme.php?name=coping%20with%20mortality

https://www.themeontology.org/theme.php?name=the%20desire%20to%20have%20children

Sounds like they are in the ballpark?

Reply:
On the immortality front, I deemed the story most similar to [Deja Q](https://themeontology.org/story.php?name=tng3x13) in which we elected to use only **coping with mortality**. I suppose it'd make sense to add the newly coined **immortality** theme to both stories though.

Good catch with **the desire to have children**. It has been added.